samples/sensor/mcp9600/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_GPIO=y
+CONFIG_SENSOR=y
+CONFIG_MCP9600=y
+CONFIG_BUILD_OUTPUT_HEX=y
```

Comment:
Should not need to specify the build output.

Reply:
Actually for the KW41Z the flash seems to prefer BIN although will work with Hex, so I put that in there to eliminate a flash warning. Guess it could disappear.